lib/ansible/modules/windows/win_chocolatey.ps1

```diff
@@ -204,6 +209,11 @@ Function Choco-Upgrade
         $cmd += " -ignoredependencies"
```

Comment:
Not an issue for you but we might want to investigate this, I believe it should be ` --ignoredependencies` with the 2nd dash